### ISSUE TITLE
Strange dissapearing elements in chromium browsers (not Firefox)

### ISSUE THREAD

**Issue by SoulDesignerTom** [locomotivemtl/locomotive-scroll#427]
Hello 👋

**Describe the bug**
The issue is showing when i'm scrolling website for more than one time. When i scroll down and then scrolling back up - so elements appears on screen with a little delay (it's empty space and then elements appear).

**To Reproduce**
1. Go to https://tomb.pl/projects/seed-cafe/
2. Scroll down to the bottom of the page
3. Scroll back up
4. See the elements appear from nowhere (some elements doesn't even display anymore).

**Expected behavior**
Elements should be displayed whole time (not dissapearing and appearing back)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser chrome
 - Version 101.0.4951.54

Thank you 👊

**Comment by Robak08** [comment]
Hi, 

I've checked website (great design btw), and it's possible that Your passing repeat: true to locomotive options object, which toggles inview class (data attribute in Your case), removing it or passing false (it's false by default) value should work for single in-transition.

**Comment by SoulDesignerTom** [comment]
Hi @Robak08 

I've got code like this:

`const scroll = new LocomotiveScroll({
        el: document.querySelector('body'),
        smooth: true
    });
	
	setTimeout(() => {
        scroll.update();
    }, 800);`

And that's pretty much true - when i remove `smooth: true` - it all works fine but i don't have this smooth scrolling effect anymore and i would love to keep it.

P.S. thanks for the design compliment - trying my best ;)

**Comment by SoulDesignerTom** [comment]
Awesome! works like a charm ;)
Thanks

**Comment by SoulDesignerTom** [comment]
I've found a better solution in different topic: https://github.com/locomotivemtl/locomotive-scroll/issues/361

just use `position:fixed;` on main element.

**Comment by Robak08** [comment]
@SoulDesignerTom 

I'm not sure (probably I'm wrong) that setting position: fixed on whole container is a good idea (maybe with simple static sites), but to resolve Your issue just change height of menu list element from height: 100% to:

```
nav.main-menu ul {
height: 100vh;
}
```

**Comment by SoulDesignerTom** [comment]
@Robak08 - thanks for info, but why do you think that `position:fixed` is not a good idea for more comples websites?
In my case it works with no issues. So just wonder...

**Comment by Robak08** [comment]
Anything beside static position is disrupting "natural" website flow. As I usually tend to lean on most simple structure. Fixed elements are positioned relative to viewport hence there might be some implications as to viewport calculations (maybe not but why overcomplicate things), as well as there might be issues with exterior elements positioning (like widgets or web components). As for statement that for more complex websites it might be not preferable - I just got feeling that it might affect negatively performance with a lot of elements transformations around viewport.
